src/views/GovernanceReviewTeam/Dates/index.tsx

```diff
@@ -0,0 +1,229 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory, useParams } from 'react-router-dom';
+import { Button } from '@trussworks/react-uswds';
+import { Field, Form, Formik, FormikProps } from 'formik';
+import { DateTime } from 'luxon';
+
+import { ErrorAlert, ErrorAlertMessage } from 'components/shared/ErrorAlert';
+import FieldErrorMsg from 'components/shared/FieldErrorMsg';
+import FieldGroup from 'components/shared/FieldGroup';
+import Label from 'components/shared/Label';
+import TextField from 'components/shared/TextField';
+import { AppState } from 'reducers/rootReducer';
+import { saveSystemIntake } from 'types/routines';
+import { SubmitDatesForm, SystemIntakeForm } from 'types/systemIntake';
+import flattenErrors from 'utils/flattenErrors';
+
+const Dates = ({ systemIntake }: { systemIntake: SystemIntakeForm }) => {
+  const { systemId } = useParams();
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const { t } = useTranslation();
+  const intakeIsLoading = useSelector(
+    (state: AppState) => state.systemIntake.isLoading
+  );
+
+  const { grtDate, grbDate } = systemIntake;
+
+  // TODO: Fix Text Field so we don't have to set initial empty values
+  const initialValues: SubmitDatesForm = {
+    grtDateDay: grtDate ? String(grtDate.day) : '',
+    grtDateMonth: grtDate ? String(grtDate.month) : '',
+    grtDateYear: grtDate ? String(grtDate.year) : '',
+    grbDateDay: grbDate ? String(grbDate.day) : '',
+    grbDateMonth: grbDate ? String(grbDate.month) : '',
+    grbDateYear: grbDate ? String(grbDate.year) : ''
+  };
+
+  const onSubmit = (values: SubmitDatesForm) => {
+    const {
+      grtDateDay,
+      grtDateMonth,
+      grtDateYear,
+      grbDateMonth,
+      grbDateDay,
+      grbDateYear
+    } = values;
+
+    const newGrtDate = DateTime.fromObject({
+      day: Number(grtDateDay),
+      month: Number(grtDateMonth),
+      year: Number(grtDateYear)
+    });
+    const newGrbDate = DateTime.fromObject({
+      day: Number(grbDateDay),
+      month: Number(grbDateMonth),
+      year: Number(grbDateYear)
+    });
+
+    const data = {
+      ...systemIntake,
+      grtDate: newGrtDate,
+      grbDate: newGrbDate,
+      id: systemId
```

Comment:
Why did you need to explicitly set the `id` here?

I'm wondering if you ran into an issue/bug and needed to manually set it.

Reply:
I think I borrowed this from elsewhere in the code where we're being strict about making sure the id is always the one that comes from the URL param. We probably don't need that here.